File: jupyter2/problems/recursion/count_paths/go/count_paths.go
package main
import "fmt"

/** begin my answer */

/** end my answer */

func main() {
	if !(countPaths(7, 3)        == 120)       { panic("wrong") }
	if !(countPathsBelow(7, 3)   == 12)        { panic("wrong") }
	if !(countPaths(12, 8)       == 125970)    { panic("wrong") }
	if !(countPathsBelow(12, 8)  == 7229)      { panic("wrong") }
	if !(countPaths(17, 13)      == 119759850) { panic("wrong") }
	if !(countPathsBelow(17, 13) == 3991995)   { panic("wrong") }
	fmt.Println("OK")
}

File: jupyter2/problems/recursion/count_paths/jl/count_paths.jl
### begin my answer

### end my answer

function main()
    @assert count_paths(7, 3)         == 120
    @assert count_paths_below(7, 3)   == 12
    @assert count_paths(12, 8)        == 125970
    @assert count_paths_below(12, 8)  == 7229
    @assert count_paths(17, 13)       == 119759850
    @assert count_paths_below(17, 13) == 3991995
    println("OK")
end

main()
